tests/functional/test_config_groups.py

```diff
@@ -260,3 +261,69 @@ def test_add_incorrect_host_to_component_group(self, cluster_with_components_on_
             component_group.host_add(test_host_1)
             assert_that_host_exists(component_group, test_host_1)
             assert_host_candidate_equal_expected(component_group.host_candidate(), [])
+
+
+class TestDeleteHostInGroups:
+    """Test deleting host related to conf group"""
+
+    @allure.step("Check that there are no hosts in conf group")
+    def check_no_hosts_in_group(self, group: GroupConfig):
+        assert len(group.hosts()) == 0, "Should not be any hosts in conf group"
+
+    def test_delete_host_from_group_after_deleting_in_cluster(self, sdk_client_fs, cluster, provider):
+        """Test that host removed from conf group after removing from cluster"""
+
+        test_host = provider.host_create(fqdn=FIRST_HOST)
+        cluster.host_add(test_host)
+        with allure.step("Create config group for cluster and add the host"):
+            cluster_group = cluster.group_config_create(name=FIRST_GROUP)
+            cluster_group.host_add(test_host)
```

Comment:
Maybe add a check that host is actually in group (maybe even make method-step) here?
If it isn't checked we won't know that deletion worked.

Reply:
added checks